tests/pages/tournamentlobbypage.test.ts

```diff
@@ -72,6 +72,21 @@ describe("Tournament lobby page", function () {
             ],
         ];
     });
+    describe("Numeric text", function () {
+        it("Value that is > as 1000 shoud be with comma", function () {
+            const tlobbyPage = new TournamentLobbyPage();
+            const numericText = tlobbyPage.numericText(6526);
+
+            expect(numericText).toEqual("6,526");
+        });
+        it("Value that is < as 1000 shoud be without comma", function () {
+            const tlobbyPage = new TournamentLobbyPage();
+            const numericText = tlobbyPage.numericText(652);
```

Comment:
There no need test that function. We should test whole scenario, if you change is really related to having comma, then we are in trouble, since it is means that change related to the language of the browser. This is means we should check at least English and Russian in browser settings. I need proof for correctness of the change. these tests does not do that.

Reply:
Still do not get it, do you want see smth like this `numericTextBinding.withCommas(joinAmount.toFixed(0), numericTextBinding.separator);` and test for withCommas function in helpers.ts ?

Reply:
yes. that should helps us control `withCommas` function output